javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+var contactForm = document.forms[0];
+
+var formControl = {};
```

Comment:
make sure that you are locally defining this variable in the function. If this form was being submitted over and over, you would be making copies of the original object and they would all have the same values!